### ISSUE TITLE
Rename local variable

### ISSUE THREAD

**Issue by smackesey** [microsoft/pylance-release#2457]
Not sure if this is a bug or purposefully unsupported feature, but attempting to rename a local variable in LSP mode appears to be a no-op:

```python
def myfunc(foo):  # Renaming does nothing
    bar = foo  # Renaming bar does nothing
    return bar
```

Running `pyright` 1.1.227 in LSP mode in Neovim.

**Comment by heejaechang** [comment]
@smackesey can you try it on vscode pylance and let us know whether it still repro? if it does, can you provide a log following instructions on https://github.com/microsoft/pylance-release/blob/main/TROUBLESHOOTING.md#filing-an-issue ?

it should give us more context to figure out what is going on. thank you.

**Comment by bschnurr** [comment]
This issue has been fixed in version 2022.3.3, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202233-23-March-2022)
